mmv1/products/accesscontextmanager/EgressPolicy.yaml

```diff
@@ -11,6 +11,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# This resource has been deprecated, please refer to ServicePerimeterEgressPolicy.
```

Comment:
Apologies, I should have been more specific. I think we should replace the description of these resources with a warning that it has been deprecated, and tell the user to go to the `google_access_context_manager_service_perimeter_*_policy` equivalent. My intention is that the registry documentation gets changed so that any users wanting to lookup the ingress/egress policy TF documentation get redirected to the correct ones.

Reply:
Actually, better yet, let's skip the docs entirely for these resources. Using `skip_docs`

Reply:
Looks like `skip_docs` is an attribute for skipping the examples. Do we have another attribute for skipping docs entirely? Otherwise I can change in the description.

Reply:
Ah, I confused that one then. It looks like we do not have an attribute. Changing the description is fine.

Reply:
Cool, updated